### DISCUSSION TITLE
Why does not q-input  set color to hint?

### DISCUSSION THREAD

**Discussion by diecamdia** [quasarframework/quasar#12120] [General - Components / Directives / etc]
Hi all,

   I have set a color on an input component q and when it is focused the color changes but the helper (hint) text message does not.

![image](https://user-images.githubusercontent.com/5762398/150326761-5de9fa45-4bf7-4a81-9a1a-8e81b5f7f1ea.png)

How can I do put the same color to helper (hint) text when the component has focus? I want set the same color and text style in cursive.

 Thanks.

**Comment by MilosPaunovic** [comment]
Use hint slot and style your text there.

**Reply by diecamdia** [reply]
Thanks for the answer, but I don't like that solution because for each input I'll probably need to apply a conditional style and then I'll have to create a variable for each of them that activates only the one with focus.

I don't know if there is another way to do it. Any suggestion is welcome.

**Comment by MilosPaunovic** [comment]
Other solution is to use CSS:
[https://codepen.io/milospaunovic/pen/vYeqXpr](https://codepen.io/milospaunovic/pen/vYeqXpr)

**Comment by diecamdia** [comment]
Hi Milos, thanks for you help:

This is what I want:
[https://codepen.io/diecamdia/pen/xxXoWEo](https://codepen.io/diecamdia/pen/xxXoWEo)

I'm thinking what I'm going to create a new component to extend q-input.

Thanks.
